scripts/sfpshow

```diff
@@ -391,6 +391,7 @@ class SFPShow(object):
                 if dump_dom:
                     sfp_type = sfp_info_dict['type']
                     dom_info_dict = state_db.get_all(state_db.STATE_DB, 'TRANSCEIVER_DOM_SENSOR|{}'.format(interface_name))
+                    dom_info_dict.update(state_db.get_all(state_db.STATE_DB, 'TRANSCEIVER_DOM_THRESHOLD|{}'.format(interface_name)))
```

Comment:
@mihirpat1 : Please validate, add UT outcome of following CLIs:

specific port (optics EEPROM/DOM)
1) show int trans eeprom Ethernet<> -n asic<>
2) show int trans eeprom --dom Ethernet<> -n asic<>

all ports where optics inserted in LC/Fixed-chassis
3) show int trans eeprom
4)show int trans eeprom --dom

Reply:
I have now added the multi asic UT enclosure to https://github.com/sonic-net/sonic-platform-daemons/pull/320.
https://github.com/sonic-net/sonic-platform-daemons/files/10157449/Unit-test_400ZR_Multiasic.txt